### PR TITLE
New source from proxyspy.net

### PR DESCRIPTION
**Pull request by vadimmarkelov**

### CODE REVIEW

File: sources/proxyspy.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+			var data = result.data;
+			var options = result.options;
+			var countriesKeys = _.keys(options.countries);
+			var anonymityCodeMap = {
+				'n': 'transparent',
+				'a': 'anonymous',
+				'h': 'elite'
+			};
+			// regex capturing groups: IP, PORT, COUNTRY, ANONYMITYLEVEL, SECURE
+			var re = /((?:\d{1,3}\.){3}\d{1,3})\:(\d+)\s+(\S{2})\-([nah])\-*([s]*)/g;
```
Comment (chill117) [inline sources/proxyspy.js]: Could you use `split` and `substr` string functions to capture this information instead? I'd rather avoid regular expressions when possible.

Comment (vadimmarkelov) [inline sources/proxyspy.js]: @chill117 yes, sure. I'll make update later this week.

Comment (chill117) [inline sources/proxyspy.js]: @vadimmarkelov Great, thanks!

**Comment by vadimmarkelov** [comment]
@chill117 i've added unit tests and removed regex as you asked

**Comment by chill117** [comment]
I almost gave up hope :) Thanks for taking the time to do this. I'll have a closer look later today.

**Pull request merged**